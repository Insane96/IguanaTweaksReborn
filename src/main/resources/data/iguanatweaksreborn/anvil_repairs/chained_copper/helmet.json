{
	"item_to_repair": {
		"id": "iguanatweaksreborn:chained_copper_helmet"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 4
		}
	]
}