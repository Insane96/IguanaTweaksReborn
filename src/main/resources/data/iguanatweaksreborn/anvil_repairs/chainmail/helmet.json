{
	"item_to_repair": {
		"id": "minecraft:chainmail_helmet"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:chain"
			},
			"amount": 4
		}
	]
}