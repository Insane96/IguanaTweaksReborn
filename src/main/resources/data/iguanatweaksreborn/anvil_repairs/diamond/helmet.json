{
	"item_to_repair": {
		"id": "minecraft:diamond_helmet"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 4
		}
	]
}