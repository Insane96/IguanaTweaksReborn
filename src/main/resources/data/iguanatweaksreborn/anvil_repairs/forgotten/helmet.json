{
	"item_to_repair": "quark:forgotten_hat",
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:leather"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:rabbit_hide"
			},
			"amount": 14,
			"max_repair": 0.5
		}
	]
}