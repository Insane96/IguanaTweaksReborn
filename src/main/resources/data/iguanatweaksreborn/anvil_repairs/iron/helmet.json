{
	"item_to_repair": {
		"id": "minecraft:iron_helmet"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:iron_nugget"
			},
			"amount": 36,
			"cost_multiplier": 9
		}
	]
}