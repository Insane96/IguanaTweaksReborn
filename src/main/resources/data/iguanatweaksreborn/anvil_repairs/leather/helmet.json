{
	"item_to_repair": "minecraft:leather_helmet",
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:leather"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:rabbit_hide"
			},
			"amount": 14,
			"max_repair": 0.5
		}
	]
}