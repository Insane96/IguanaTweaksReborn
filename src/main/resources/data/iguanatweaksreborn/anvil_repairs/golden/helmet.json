{
	"item_to_repair": {
		"id": "minecraft:golden_helmet"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 36,
			"cost_multiplier": 9
		}
	]
}